### PR TITLE
Update README and Gradle version

### LINKED ISSUE #17: Move to jitpack for publishing the library

### PR DESCRIPTION
**Pull request by tevjef**
Resolves #17 and #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,17 +9,42 @@ A nice swipe layout that provides new actions with a material design look and fe
...
+    maven { url 'https://jitpack.io' }
+  }
+}
+```
+
+Step 2. Add the dependency
+
+```groovy
+dependencies {
+  implementation 'com.github.prolificinteractive.patrons:patron:${patronsVersion}'
```
Comment (quentin41500) [inline README.md]: so in your case: `implementation 'com.github.prolificinteractive:Chandelier:${chandelierVersion}'`

File: README.md [hunk 2]
```diff
@@ -9,17 +9,42 @@ A nice swipe layout that provides new actions with a material design look and fe
...
+
+  // If you want patrons with encryption of your shared preferences
+  implementation 'com.github.prolificinteractive.patrons:patron-conceal:${patronsVersion}'
```
Comment (quentin41500) [inline README.md]: remove this and comment

File: README.md [hunk 3]
```diff
@@ -9,17 +9,42 @@ A nice swipe layout that provides new actions with a material design look and fe
...
+}
+```
+
+
 Usage
 -----
 
-1. Add `compile 'com.prolificinteractive:chandelier:0.1.0'` to your dependencies.
+1. Add `implementation 'com.prolificinteractive:chandelier:0.1.0'` to your dependencies.
```
Comment (quentin41500) [inline README.md]: We could just remove this now that we have installation.

File: gradle.properties [hunk 4]
```diff
@@ -17,21 +17,18 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
```
Comment (quentin41500) [inline gradle.properties]: I think we can remove this file now that we are using jitpack

File: settings.gradle [hunk 5]
```diff
@@ -1 +1,3 @@
 include ':app', ':library'
+
+rootProject.name = 'chandelier'
```
Comment (quentin41500) [inline settings.gradle]: Could you try adding that to the module directly?

Comment (tevjef) [reply settings.gradle]: Everything looks good. 

https://jitpack.io/#prolificinteractive/chadelier

**Review by quentin41500 — APPROVED** [review]
👍 LGTM

**Pull request merged**